gwcs/coordinate_frames.py

```diff
@@ -471,7 +471,8 @@ def coordinates(self, *args):
 
     def coordinate_to_quantity(self, *coords):
         if isinstance(coords[0], time.Time):
-            if self.reference_frame.value:
+            ref_value = self.reference_frame.value
+            if not isinstance(ref_value, np.ndarray):
                 return (coords[0] - self.reference_frame).to(self.unit[0])
             else:
```

Comment:
Can you explain the changes here and in conftest.py as they relate to fix_inputs? I can't see the connection. If there isn't any they probably should be explained in the PR

Reply:
The conftest changes add a pytest fixture used in testing fixed_inputs.